share/eds/retriever_test.go

```diff
@@ -124,3 +129,56 @@ func TestRetriever_MultipleRandQuadrants(t *testing.T) {
 	_, err = ses.Reconstruct(ctx)
 	assert.NoError(t, err)
 }
+
+func TestFraudProofValidation(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*15)
+	defer t.Cleanup(cancel)
+	bServ := mdutils.Bserv()
+
+	var errByz *byzantine.ErrByzantine
+	faultHeader, err := generateByzantineError(ctx, t, 2, bServ)
```

Comment:
can you test with a larger ODS too? LIke make this table test

Reply:
Done